packages/headless/src/controllers/core/facets/facet-search/facet-search.ts

```diff
@@ -22,6 +19,15 @@ export interface GenericFacetSearchProps<T extends FacetSearchState> {
   options: FacetSearchOptions;
   getFacetSearch: () => T;
   isForFieldSuggestions: boolean;
+
+  executeFacetSearchActionCreator: (
+    facetId: string
+  ) => AsyncThunkAction<unknown, string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+  executeFieldSuggestActionCreator: (facetId: string) => AsyncThunkAction<
```

Comment:
Do we name our actionCreator potatoActionCreator usually in the rest of the code?

Reply:
Yes